x-pack/test/api_integration/apis/maps/index_data.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+
+  describe('index feature data', () => {
+    it('should add data to an existing index', async () => {
```

Comment:
can we add a stub here for a success with an actual GeoJson.geometry into a `geo_shape` field?